src/lm.jl

```diff
@@ -72,9 +72,9 @@ function fit{LmRespT<:LmResp,LinPredT<:LinPred}(::Type{LinearModel{LmRespT,LinPr
     LinearModel(rr, pp)
 end
 function fit(::Type{LinearModel}, X::Matrix, y::Vector)
-    T = promote_type(eltype(X), eltype(y), Float32)
+    T = promote_type(eltype(X), eltype(y), Float64)
```

Comment:
I'm not sure what the original code is supposed to do exactly, but `yy` isn't used anywhere (except for its type, which is a waste). Why not simply use `float` as for GLMs? In general, I'd keep the two model types completely in sync unless there's a good reason not to.